plugins/devo/help.md

```diff
@@ -183,7 +183,8 @@ _This plugin does not contain any custom output types._
 
 # Version History
 
-* 3.0.0 - Fix an issue in Get New Alerts trigger to filter duplicate alerts. Output contains list of new alerts
+* 3.0.1 - Fix an issue in Get New Alerts trigger to filter duplicate alerts
+* 3.0.0 - Output contains list of new alerts
```

Comment:
The changelog should remain static - we shouldn't be revising earlier versions of changelog messages

Reply:
Sure, we updated that one.